### ISSUE TITLE
Error when trying to double scene which root's node contains script

### ISSUE THREAD

**Issue by p10tr3k** [bitwes/Gut#524]
If you try to double scene which root element contains script you will have error like this:
```
Parser Error: Cannot call the parent class' virtual function "_init()" because it hasn't been defined.
```

Example: 
test file.gd
```
extends GutTest

func test_load_scene():
	var Ship = load("res://ship.tscn")
	var ship_double = double(Ship)
```
ship.gd 
```
extends Node2D
```
Godot: v4.1.3.stable.official [f06b6836a]
Gut Version: 9.1.0
MacOS

Minimal project:
[GutBug.zip](https://github.com/bitwes/Gut/files/13404562/GutBug.zip)

**Comment by p10tr3k** [comment]
Strange, this also gives the same error:
```
double(Node2D).new()
```

**Comment by bitwes** [comment]
Fixed in 9.1.1 (not released yet).
I think this was fixed by "* Fixed numerous issues with doubling that were caused by the port from 3.x.  Most of these involved using the INCLUDE_NATIVE doubling strategy." (from CHANGES.md)
